conda/core/subdir_data.py

```diff
@@ -608,12 +610,12 @@ def read_mod_and_etag(path):
                 match_objects = take(3, re.finditer(REPODATA_HEADER_RE, m))
                 result = dict(map(ensure_unicode, mo.groups()) for mo in match_objects)
                 return result
-        except (BufferError, ValueError, OSError):  # pragma: no cover
+        except (BufferError, ValueError):  # pragma: no cover
```

Comment:
This error was found by the type checker. OSError is handled in the next clause. IOError is a subclass of OSError, so it doesn't make sense to catch (IOError, OSError)

Reply:
Nice catch!